__tests__/app.test.js

```diff
@@ -69,6 +69,24 @@ describe("GET /api/articles/:article_id", ()=>{
       })
     })
   })
+  test("200: Responds Responds with an article object with the specified article_id including comment_count", ()=>{
+    return request(app)
+    .get("/api/articles/9")
+    .expect(200)
+    .then(({body:{article}})=>{
+      expect(article).toMatchObject({
+        article_id: 9,
+        title: "They're not exactly dogs, are they?",
+        topic: 'mitch',
+        author: "butter_bridge",
+        body: "Well? Think about it.",
+        created_at: "2020-06-06T09:10:00.000Z",
+        votes: 0,
+        article_img_url:  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        comment_count: 2
```

Comment:
Rather than amending this test, it would be better if you could do a new one to show that comment_count has now been added. Your previous tests should always be kept as they are. 👍 
If this one suddenly starts to fail, then have a think about how you could use a more flexible matcher 🙂